Makefile

```diff
@@ -63,7 +63,7 @@ run: my.env .docker-build
 
 .PHONY: shell
 shell: my.env .docker-build
-	${DC} run base bash
+	${DC} run web bash
```

Comment:
This should run in the `web` container since that has the environment variables we need.